src/main/java/org/gridsuite/loadflow/server/LoadFlowParametersController.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.service.parameters.LoadFlowParametersService;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + LoadFlowApi.API_VERSION + "/parameters")
+@Tag(name = "LoadFlow parameters")
+public class LoadFlowParametersController {
+
+    private final LoadFlowParametersService parametersService;
+
+    public LoadFlowParametersController(LoadFlowParametersService parametersService) {
+        this.parametersService = parametersService;
+    }
+
+    @PostMapping(value = "", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create parameters")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "parameters were created")})
+    public ResponseEntity<UUID> createParameters(
+            @RequestBody LoadFlowParametersInfos parametersInfos) {
+        return ResponseEntity.ok().body(parametersService.createParameters(parametersInfos));
+    }
+
+    @PostMapping(value = "/default")
+    @Operation(summary = "Create default parameters")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Default parameters were created")})
+    public ResponseEntity<UUID> createDefaultParameters() {
+        return ResponseEntity.ok().body(parametersService.createDefaultParameters());
+    }
+
+    @PostMapping(value = "/{sourceParametersUuid}")
+    @Operation(summary = "Duplicate parameters")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "parameters were duplicated")})
+    public ResponseEntity<UUID> duplicateParameters(
+            @Parameter(description = "source parameters UUID") @PathVariable("sourceParametersUuid") UUID sourceParametersUuid) {
+        return parametersService.duplicateParameters(sourceParametersUuid)
+                .map(duplicatedParametersUuid -> ResponseEntity.ok().body(duplicatedParametersUuid))
+                .orElse(ResponseEntity.notFound().build());
```

Comment:
[`ResponseEntity.of(Optional<T> body)`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html#of(java.util.Optional)) 😉
```suggestion
        return ResponseEntity.of(parametersService.duplicateParameters(sourceParametersUuid));
```